bundles/org.openhab.binding.siemensrds/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -163,7 +163,7 @@
 	</channel-type>
 
 	<channel-type id="roomHumidity">
-		<item-type unitHint="%">Number:Dimensionless</item-type>
+		<item-type>Number:Dimensionless</item-type>
```

Comment:
Please revert:
```suggestion
		<item-type unitHint="%">Number:Dimensionless</item-type>
```

Reply:
Dont't think so as the state pattern does nto have %unit% but is fixed to a percentage already.